apps/web/src/store/actions/trackActions.js

```diff
@@ -0,0 +1,327 @@
+import { AnalyticsEvents } from '@hylo/shared'
+
+export const MODULE_NAME = 'Tracks'
+export const CREATE_TRACK = `${MODULE_NAME}/CREATE_TRACK`
+export const ENROLL_IN_TRACK = `${MODULE_NAME}/ENROLL_IN_TRACK`
+export const ENROLL_IN_TRACK_PENDING = `${MODULE_NAME}/ENROLL_IN_TRACK_PENDING`
+export const FETCH_TRACK = `${MODULE_NAME}/FETCH_TRACK`
+export const LEAVE_TRACK = `${MODULE_NAME}/LEAVE_TRACK`
+export const LEAVE_TRACK_PENDING = `${MODULE_NAME}/LEAVE_TRACK_PENDING`
+export const UPDATE_TRACK = `${MODULE_NAME}/UPDATE_TRACK`
+export const UPDATE_TRACK_PENDING = `${MODULE_NAME}/UPDATE_TRACK_PENDING`
+export const UPDATE_TRACK_ACTION_ORDER = `${MODULE_NAME}/UPDATE_TRACK_ACTION_ORDER`
+export const UPDATE_TRACK_ACTION_ORDER_PENDING = `${MODULE_NAME}/UPDATE_TRACK_ACTION_ORDER_PENDING`
+
+const CommentFieldsFragment = `
```

Comment:
Could we re-use an existing fragment here? 

'@hylo/graphql/fragments/commentFieldsFragment' looks almost identical?

Reply:
probably for the CommentFields yes